src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
I had the same confusion about why `msFullscreenElement` was being (re)checked but then I [read the docs](https://msdn.microsoft.com/en-us/library/dn254937\(v=vs.85\).aspx) and I assume this variable is there but `null` if we're not *currently* in fullscreen. Their docs aren't very clear but checking on the IE11 console that seems to be the case.

> It can even store the `window.top !== window.self` bit

Hmm, I wonder if we need to handle the case where a jQuery in one frame is manipulating elements in another frame. We do get occasional bug reports about those use cases. If we wanted to handle that we would need to use `elem.ownerDocument` and `elem.document.defaultView` each time?